server/podman.yaml

```diff
@@ -0,0 +1,6 @@
+podman:
```

Comment:
This would be:
```
podman:
   command: podman
    # init_command: \{command} .. (optional or default)
    start_command: \{command} run --name {name} -d {image}
    stop_command: \{command} stop {name}
    pull_command: \{command} pull {image} 

```

Reply:
Updated the podman.yaml